templates/firewall/resources/vkcs_networking_secgroup_rule.md.tmpl

```diff
@@ -13,6 +13,12 @@ description: |-
 ## Example Usage
 {{tffile .ExampleFile}}
 
+## Rule for all port udp protocol
+{{tffile "examples/firewall/networking_secgroup_rule/main-all-ports-protocol.tf"}}
+
+## Rule for one port tcp protocol
```

Comment:
i believe this must be the first example

Reply:
changed